Source/Scene/Model.js

```diff
@@ -539,6 +539,10 @@ define([
         this.clippingPlanes = options.clippingPlanes;
         // Used for checking if shaders need to be regenerated due to clipping plane changes.
         this._clippingPlanesState = 0;
+        // If defined, it's used to position the clipping planes instead of the modelMatrix.
+        // This is so that when models are part of a tileset they all get clipped relative
+        // to the root tile.
+        this._clippingPlaneOffsetMatrix = undefined;
```

Comment:
Slight wording tweak `// If defined, use this matrix to position the clipping planes instead of the modelMatrix.`